include/robot_interfaces/opencv_driver.hpp

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <ctime>
+#include <iostream>
+
+#include <real_time_tools/process_manager.hpp>
+#include <real_time_tools/thread.hpp>
+#include <real_time_tools/threadsafe/threadsafe_timeseries.hpp>
+#include <real_time_tools/timer.hpp>
+#include "opencv2/opencv.hpp"
+
+namespace robot_interfaces
+{
+/**
+ * @brief Driver for interacting with any camera using OpenCV.
+ */
+
+template <typename OpenCVObservation>
+class OpenCVDriver
+{
+public:
+    cv::VideoCapture video_capture;
```

Comment:
In C++ class variables need to be suffixed with `_`.
```suggestion
    cv::VideoCapture video_capture_;
```